amundsen_application/api/v0.py

```diff
@@ -4,18 +4,30 @@
 from flask import current_app as app
 from flask.blueprints import Blueprint
 
+from amundsen_application.api.metadata.v0 import USER_ENDPOINT
+from amundsen_application.api.utils.request_utils import request_wrapper
 from amundsen_application.models.user import load_user
 
+REQUEST_SESSION_TIMEOUT = 10
+
 LOGGER = logging.getLogger(__name__)
 
 blueprint = Blueprint('api', __name__, url_prefix='/api')
 
 
 @blueprint.route('/auth_user', methods=['GET'])
 def current_user() -> Response:
-    if (app.config['AUTH_USER_METHOD']):
-        user = app.config['AUTH_USER_METHOD'](app)
+    if app.config['AUTH_USER_METHOD']:
+        user_info = app.config['AUTH_USER_METHOD'](app)
     else:
-        user = load_user({'user_id': 'undefined', 'display_name': '*'})
+        user_info = {'email': 'test@test.com', 'user_id': 'test'}
```

Comment:
should we just throw exception if the user is not authenticated?

Reply:
That's a good point.

Maybe this was just something that was there since the beginning and never got cleaned up. I wonder if there are any instances where we want to test something without going through auth.

Reply:
The default open source version of Amundsen doesn't use auth, and there's a change other's wont have any auth configured in their development environments. 

Throwing an exception is fine, but if we remove this line we'd need to make sure the pieces of the application that leverage this information will work as expected if `email` and `user_id` are missing -- I found it easier to add in the required `User` fields here at the start to avoid chasing down potential bugs, knowing that in reality people would configure auth for a real use case.